tests/test.sh

```diff
@@ -1,12 +1,18 @@
-set -e 
+set -e
 
-export ADAREDUCER_TEMP_DIR=` mktemp -d` 
+# Make a temporary directory
+export ADAREDUCER_TEMP_DIR=` mktemp -d`
```

Comment:
Is there a reason for the leading space in front of `mktemp`?

Reply:
Not particularly. Fixed!